test/index.html

```diff
@@ -155,6 +160,46 @@ <h1>Some Sample CFPB Charts</h1>
                      data-chart-type="bar">
                      This is the chart description.
                 </div>
+                <hr>
+                <div class="cfpb-chart"
+                     data-chart-color="navy"
+                     data-chart-description="Indexed number of people with inquiries."
+                     data-chart-source="consumer-credit-trends/auto-loans/inquiry_rate_AUT.csv"
+                     data-chart-title="Inquiry rate auto"
```

Comment:
Can we use something closer to the actual chart title here, or change the name of the variable? Same change for the other inquiry charts that follow.

Reply:
Hmm, this is actually only used in the TileMap. We should nix it altogether from the other chart demos.

Reply:
Actually, doesn't appear to even be used there. Opened https://github.cfpb.gov/CFGOV/platform/issues/2978